### ISSUE TITLE
5-column widget layout for podcasts section

### ISSUE THREAD

**Issue by MirandaEcho** [INN/umbrella-borderzine#45]

**Comment by joshdarby** [comment]
@MirandaEcho Are the posts in this section supposed to show media play icons like on Workday and City Limits?

**Comment by MirandaEcho** [comment]
@joshdarby yeah, lets have the link be an icon that links to the podcast page, like we did for Workday.
